engine-tests/src/tests/xcc.rs

```diff
@@ -158,6 +159,90 @@ fn test_xcc_eth_gas_cost() {
     );
 }
 
+#[test]
+fn test_xcc_external_fund() {
+    // In this test we intentionally do not bridge wNEAR into the Engine.
+    // The purpose of the `fund_xcc_sub_account` functionality is to allow using
+    // the XCC feature in an Engine instance where there is no bridged wNEAR.
+
+    // Set up Engine contract
+    let aurora = deploy_evm();
+    let chain_id = AuroraRunner::default().chain_id;
```

Comment:
All the tests for XCC use `near-sdk-sim` (will eventually be replaced with `aurora-workspace`) because they require resolving promises. So the `aurora` instance here is actually [AuroraAccount](https://github.com/aurora-is-near/aurora-engine/blob/56f5c90857edf48252c24cbd2d2f93fb582ff235/engine-tests/src/tests/state_migration.rs#L77) not `AuroraRunner`. The former does not include a `chain_id` field. I suppose we could add that field, but I don't want to put additional effort into the code we have `near-sdk-sim` when it is being replaced.